tasks/js-basics.md

```diff
@@ -55,8 +55,10 @@ When you finish this task you can proceed to the next one.
 ## Extra materials
 
 - [Курс "Введение в программирование" от Hexlet](https://ru.hexlet.io/courses/introduction_to_programming)
+- [FunFunFunction - Functional programming in JavaScript](https://www.youtube.com/playlist?list=PL0zVEGEvSaeEd9hlmCXrk5yUyqUag-n84)
```

Comment:
Do we really need those links?

Reply:
FunFunFunction - has a very good explanation of the general principles of functional programming, so i think it was useful for newcomers